packages/app/navigation/linking.ts

```diff
@@ -70,6 +71,28 @@ const linking: LinkingOptions<ReactNavigation.RootParamList> = {
 
     return getStateFromPath(finalPath, config);
   },
+  async getInitialURL() {
+    const url = await Linking.getInitialURL();
+    const initialURL = url ? new URL(url) : null;
+    if (initialURL && handleResponse(initialURL)) {
+      // URL handled by Wallet Mobile SDK
+      return null;
+    } else {
+      return url;
+    }
+  },
+  subscribe(listener) {
+    const linkingSubscription = Linking.addEventListener("url", ({ url }) => {
+      const handledByMobileSDK = handleResponse(new URL(url));
+      if (!handledByMobileSDK) {
+        listener(url);
+      }
+    });
+
+    return function cleanup() {
+      linkingSubscription.remove();
+    };
+  },
```

Comment:
Listens for incoming deeplinks and passes the URL to the `handleResponse` function to be handled by the mobile sdk